portfolio/src/main/webapp/index.html

```diff
@@ -73,6 +73,11 @@ <h2>Fact Hat</h2>
             </div>
         </div>
         <button id="page_button" onclick="window.location.href='/images.html'"><strong>Go to Gallery!</strong></button>
+        <div id="section">
```

Comment:
do you want to use class instead of id?

Reply:
I changed section to use class instead of id. Because I changed the other classes and ids in another pull request I only changed the id for section which is part of the changed lines for this pull request.